js/App.js

```diff
@@ -0,0 +1,68 @@
+var Socket = require('./utils/Socket'),
```

Comment:
Not sure I totally understand the question.

We could use it wherever we needs what it contains.

If you're asking if I integrated the socket view to pull the port it should connect to from the Server Config model, then yes I did.